docs/reference/metadata.md

```diff
@@ -194,13 +190,13 @@ Subfields:
 
 A `person_or_org` is described with the following subfields:
 
-| Field | Cardinality |   Description   |
-|:-----:|:-----------:|:----------------|
-| ``name`` | (0 if `type` is `personal` / 1 if `type` is `organisational`) | The full name of the organisation. For a person this field is generated from `given_name` and `family_name` |
+| Field            | Cardinality |   Description   |
+|:---------------------:|:-----------:|:----------------|
+| ``name``              | (0 if `type` is `personal` / 1 if `type` is `organisational`) | The full name of the organisation. For a person this field is generated from `given_name` and `family_name` |
 | ``type`` | (0-1, CV) | The type of name. Either ``personal`` or ``organisational``. |
 | ``role`` | (0-1, CV) | The role of the person or organisation selected from a customizable controlled vocabulary. |
-| ``given_name``  | (1 if `type` is `personal` / 0 if `type` is `organisational`) | Given name(s) if the type is a personal name. |
-| ``family_name`` | (1 if `type` is `personal` / 0 if `type` is `organisational`) | Family name if type is a personal name. |
+| ``given_name`` | (1 if `type` is `personal` / 0 if `type` is `organisational`) | Given name(s) if the type is a personal name. |
+| ``family_name`` &nbsp; | (1 if `type` is `personal` / 0 if `type` is `organisational`) | Family name if type is a personal name. |
```

Comment:
This `&nbsp;` is a bit of a hack to fix the look of the "Field" column's content in the table:

Before:

![docs_broken_lines_fields](https://user-images.githubusercontent.com/1932651/116618924-620ecd80-a905-11eb-865f-96d671790460.png)

After

![docs_fixed_lines_fields](https://user-images.githubusercontent.com/1932651/116618937-6804ae80-a905-11eb-9692-2dd07b0224f0.png)